src/main/java/org/apache/commons/collections4/bloomfilter/hasher/HashFunctionValidator.java

```diff
@@ -20,7 +20,7 @@
 /**
  * Contains validation for hash functions.
  */
-final class HashFunctionValidator {
+public final class HashFunctionValidator {
```

Comment:
Why make this public here? That would fix it in the API. Currently it is only used internally.